updating/updating-restricted-network-cluster/restricted-network-update-osus.adoc

```diff
@@ -98,20 +98,22 @@ include::modules/update-service-configure-cvo.adoc[leveloffset=+3]
 See xref:../../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object[Enabling the cluster-wide proxy] to configure the CA to trust the update server.
 ====
 
-[id="update-service-delete-service"]
-== Deleting an OpenShift Update Service application
+[id="next-steps_updating-restricted-network-cluster-osus"]
+== Next Steps
 
-You can delete an OpenShift Update Service application by using the {product-title} web console or CLI.
+Before updating your cluster, you should have performed the following tasks:
```

Comment:
ISG: 
**should** _verb_
Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed. For example, when parameters must be reset, write “Reset the parameters” and not “You should reset the parameters”.

This section looks like a prerequisites section, but it's more of a checklist of things you should've accomplished throughout this assembly, IIUC. I understand the utility of it, but I would vote for rewriting it so that it is more straightforward. You could use the phrasing from the prereqs earlier in this assembly for consistency (the existing content uses "mirrored to," for example). 

Optional rewrite suggestion:

```
Before updating your cluster, confirm that the following conditions are met:

* The Cluster Version Operator (CVO) is configured to use your locally-installed OpenShift Update Service application.
* The release image signature config map for the new release is applied to your cluster.
* The current release and update target releases are mirrored to a locally accessible registry.
* A recent graph data container image has been mirrored to your local registry.

After you configure your cluster to use the locally-installed OpenShift Update Service and local mirror registry, you can use any of the following update methods:

** xref:../../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster using the web console]
** xref:../../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI]
** xref:../../updating/preparing-eus-eus-upgrade.adoc#preparing-eus-eus-upgrade[Preparing to perform an EUS-to-EUS update]
** xref:../../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools[Performing a canary rollout update]
** xref:../../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes RHEL compute machines]
```

Reply:
Great, thanks for the suggestion! 

That checklist is basically as you say: a "prereqs" list that isn't explicitly a prereq list, and really it just makes sure that users actually followed all the steps in the assembly before jumping to any of the linked update pages.

I was struggling to think of how to introduce the list without a prereq heading, so your suggestion is super helpful. Thanks!